jerry-debugger/jerry-client-ts/src/lib/protocol-handler.ts

```diff
@@ -616,6 +646,15 @@ export class JerryDebugProtocolHandler {
       throw new Error('attempted resume while not at breakpoint');
     }
     this.lastBreakpointHit = undefined;
+    let cmd = 'Next';
+    if (code === SP.JERRY_DEBUGGER_STEP) {
+      cmd = 'Step';
+    } else if (code === SP.JERRY_DEBUGGER_FINISH) {
+      cmd = 'Finish';
+    } else if (code === SP.JERRY_DEBUGGER_CONTINUE) {
+      cmd = 'Continue';
+    }
+    this.logSentPacket(cmd);
```

Comment:
Maybe create a `logSentCommand(cmd)` helper to keep this method here a bit cleaner? Half the lines are about logging now.